test/e2e/support/utils/login.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Internal dependencies
+ */
+import { pressWithModifier } from './press-with-modifier';
+import { META_KEY } from './';
+
+const WP_ADMIN_USER = {
```

Comment:
Can we move this logic to its own file and export `WP_ADMIN_USER`, `WP_USERNAME` and `WP_PASSWORD` from there to be able to share in all test helpers?

Reply:
Covered in this commit:
https://github.com/WordPress/gutenberg/pull/11701/commits/384afa4fbdc80af4ecced9dc0623e0b178b4038f